samples/Jupyter-Notebook/JupyterNotebook.ipynb

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
The directory in `samples/` needs to match exactly the name for the language in `languages.yml` ;)